externs/olx.js

```diff
@@ -377,6 +380,35 @@ olx.OverlayOptions.prototype.insertFirst;
 
 
 /**
+ * If set to `true` the map is panned when calling `setPosition`, so that the
+ * overlay is entirely visible in the current viewport.
+ * The default is `true`.
+ * @type {boolean|undefined}
+ * @api
+ */
+olx.OverlayOptions.prototype.autoPan;
+
+
+/**
+ * The options used to create a `ol.animation.pan` animation. This animation
+ * is only used when `autoPan` is enabled. If set to `null` the panning is not
+ * animated.
```

Comment:
I think the docs should mention the default behavior, that is when `autoPanAnimation` is not defined at all.